modules/mobile/app/controllers/mobile/v0/maintenance_windows_controller.rb

```diff
@@ -24,10 +24,13 @@ class MaintenanceWindowsController < ApplicationController
         %i[lighthouse disability_rating],
         %i[lighthouse letters_and_documents],
         %i[lighthouse immunizations],
```

Comment:
So there are four mhv related maintenance windows we're looking for: mhv, mhv platform, mhv meds, and mhv sm. If mhv or mhv platform is down, then rx and secure messaging are down.